src/session.js

```diff
@@ -49,6 +50,7 @@ export default class Session extends EventEmitter {
         return mustache.render(TASK_TEMPLATE, {
             cookie:               cookies.replace(/'/g, "\\'"),
             sessionId:            this.id,
+            isFirstPageLoad:      this.pageLoadCount++ === 0,
```

Comment:
Avoid using unary operators which modify state within binary expressions (I wonder if we have eslint rule for this). So it should be:
```js
this.pageLoadCount++;
...
isFirstPageLoad: this.pageLoadCount === 1
``` 
Actually, it's my bad, AFAIR it was my suggestion to use comparison with 0. This variant looks semantically correct.